styles/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+    background-color: rgb(211, 210, 210);
+}
+
+.patrat {
+    position: relative;
+    height: 300px;
+    width: 300px;
+}
+
+.patrat1 {
+    background-color: yellow;
+}
+
+.patrat2 { 
+    top: 150px;
```

Comment:
Aici te poti folosi de faptul ca patratul 2 este in interiorul patratului 1 si poti pune top:50% si left:50%. Astfel ar fi fix la jumatatea patratului 1